### ISSUE TITLE
Create text chat feature

### ISSUE THREAD

**Issue by capnmidnight** [capnmidnight/Calla#68]
For a variety of reasons, including accessibility and conversation logging, there should be

- [ ] an area on-screen for typing in messages. 
- [ ] an optional text-to-speech engine to read the messages as they come in.
  - [ ] option to change voice parameters.
  - [ ] bonus points for getting the speech spatialized.
- [ ] an options speech-to-text recognition to send spoken words to the chat window.
  - [ ] an option to hide speech-to-text from the chat window.

**Comment by capnmidnight** [comment]
archiving
